activemodel/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+*   Remove unused `method_name` from `AttributeMethodMatch`
```

Comment:
Can you remove the changelog entry? It is not needed since this class is private.

Reply:
Sure will do.

Reply:
Removed.